proj_circle.yml

```diff
@@ -47,6 +47,20 @@ test:
   post:
     - pipenv run coverage html -d $CIRCLE_ARTIFACTS
 
+deployment:
+  staging:
+    branch: develop
+    commands:
+      - "[[ ! -s \"$(git rev-parse --git-dir)/shallow\" ]] || git fetch --unshallow"
+      - git push git@heroku.com:{{project_name}}staging.git $CIRCLE_SHA1:refs/heads/master
```

Comment:
Shouldn't the Heroku app name be following the same pattern here as in production? I mean, `{{project_name}}-staging` instead of `{{project_name}}staging`.

Reply:
@laisvarejao

Reply:
@chocoelho you're right! it's a typo! I'll fix it, thank you!